pom.xml

```diff
@@ -456,12 +456,20 @@
                 <configuration>
                     <!-- 略过整个单元测试的执行，不推荐 -->
                     <skip>${skipTests}</skip>
+                    <argLine>-Xms690m -Xmx690m -XX:MaxPermSize=512m -noverify</argLine>
```

Comment:
Do we need that config?

Reply:
If we do not have this parameter, the mvn install command will be prompted.

```java
[INFO] 
[INFO] --- maven-javadoc-plugin:2.10.3:jar (default) @ soul-bootstrap ---
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:2.10.3:javadoc' has not been previously called for the module: 'org.dromara:soul-spring-boot-starter-gateway:jar:2.2.1'. Trying to invoke it...
[ERROR] Error occurred during initialization of VM, trying to use an empty MAVEN_OPTS...
[ERROR] MavenInvocationException: Error occurred during initialization of VM, try to reduce the Java heap size for the MAVEN_OPTS environnement variable using -Xms:<size> and -Xmx:<size>.
[WARNING] Creating fake javadoc directory to prevent repeated invocations: /Users/zeyangg/SynologyDrive/ww/soul/soul-spring-boot-starter/soul-spring-boot-starter-gateway/target/apidocs
Creating fake javadoc directory to prevent repeated invocations: /Users/zeyangg/SynologyDrive/ww/soul/soul-spring-boot-starter/soul-spring-boot-starter-gateway/target/apidocs


[ERROR] Error fetching link: /Users/zeyangg/SynologyDrive/ww/soul/soul-spring-boot-starter/soul-spring-boot-starter-gateway/target/apidocs/package-list. Ignored it.
[INFO] 
Loading source files for package org.dromara.soul.bootstrap...
Loading source files for package org.dromara.soul.bootstrap.configuration...
Loading source files for package org.dromara.soul.bootstrap.filter...
Constructing Javadoc information...
```